autometa/common/kmers.py

```diff
@@ -495,23 +498,25 @@ def embed(
     if isinstance(kmers, str) and os.path.exists(kmers) and os.path.getsize(kmers):
         try:
             df = pd.read_csv(kmers, sep="\t", index_col="contig")
-        except ValueError:
+        except (ValueError, TypeError):
```

Comment:
Does `TypeError` get raised from `pd.read_csv(...)`?

Reply:
Yeah, I encountered this when I was passing in a Mock or MagicMock object to it. It seems it is raised when the input file is not a correct object

Reply:
See comment above